packages/app-layout/test/visual/lumo/app-layout.test.js

```diff
@@ -8,7 +9,7 @@ describe('app-layout', () => {
 
   beforeEach(() => {
     div = document.createElement('div');
-    div.style.height = '100%';
+    div.style.height = '600px';
```

Comment:
Could we achieve the same by explicitly setting the viewport size? 

`await setViewport({ width: 1024, height: 600 });`

Reply:
Or even globally by specifying `viewportSize` here:

https://github.com/vaadin/web-components/blob/3ee76509f5517824a3750f6d0e7a62016b3e74c5/wtr-utils.js#L270-L275

Reply:
Yes, ideally we would use `setViewport()` but it needs to be implemented: https://github.com/modernweb-dev/web/issues/1072
I'll look into that since it should be easy (especially when using `concurrency: 1` and single session).